### PR TITLE
Lab8

### PR DESCRIPTION
**Pull request by PassKeyRa**

### CODE REVIEW

File: monitoring/grafana/config.monitoring [hunk 1]
```diff
@@ -0,0 +1,3 @@
+GF_SECURITY_ADMIN_USER=admin
+GF_SECURITY_ADMIN_PASSWORD=gO/JrMlJorSkUJcg
```
Comment (chermnyx) [inline monitoring/grafana/config.monitoring:2]: Credentials in git

File: monitoring/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+x-logging: &x-logging
+  logging:
+    driver: "json-file"
+    options:
+      max-size: "100m"
+      max-file: "10"
+      tag: "{{.ImageName}}|{{.Name}}"
+
+services:
```
Comment (chermnyx) [inline monitoring/docker-compose.yml:11]: No healthchecks

**Review by chermnyx — APPROVED** [review]

File: monitoring/docker-compose.yml (hunk 2 above)
Comment (PassKeyRa) [reply monitoring/docker-compose.yml:11]: Without bonus

File: monitoring/grafana/config.monitoring (hunk 1 above)
Comment (PassKeyRa) [reply monitoring/grafana/config.monitoring:2]: Yup, do not use them in production. But of course, it should environment variables

**Review by boggda — APPROVED** [review]

**Pull request closed without merging**